app/services/hts_service/reports/pepfar/hts_tst_fac1.rb

```diff
@@ -68,8 +77,10 @@ def query
         AND access_type.value_coded = #{concept('Health facility').concept_id}
         INNER JOIN obs hiv_status ON hiv_status.person_id = person.person_id
         AND hiv_status.concept_id = #{concept('HIV status').concept_id}
+        LEFT JOIN obs pregnancy_status ON pregnancy_status.person_id = person.person_id
+        AND pregnancy_status.concept_id = #{concept('Pregnancy status').concept_id}
```

Comment:
Add the voided filter on all joins. This will fetch everything. Otherwise this looks okay